thirdparties/glog/glog.patch

```diff
@@ -716,63 +725,91 @@ index 0000000..1f3c8c1
 +// under the License.
 +
 +#include "logging_async.h"  // NOLINT
++#include "utilities.h"
 +
 +#include <iostream>
 +
 +namespace google {
 +
 +AsyncLogger::AsyncLogger(google::base::Logger* logger, uint32_t maxBufferSize)
 +    : logger_(logger),
-+      logger_thread_(),
 +      maxBufferSize_(maxBufferSize),
 +      appThreadsBlockedCount_(0),
 +      flushCount_(0),
-+      mutex_(),
 +      wakeFluchCV_(),
 +      freeBufferCV_(),
 +      flushCompleteCV_(),
 +      activeBuffer_(new Buffer()),
 +      flushBuffer_(new Buffer()),
-+      running_(false) {}
++      running_(false),
++      current_(0) {}
 +
 +AsyncLogger::~AsyncLogger() {
 +    if (running_) {
 +        Stop();
 +    }
 +}
 +
++// NOTE: this function only run once in child process
```

Comment:
the whole thing between line 763 and line 775 isn't atomic, it's better to encapsulate them into std::call_once

Reply:
> 

yep, i will improve it.